Classes/Base.php

```diff
@@ -278,4 +278,30 @@ public static function logout()
 			$session->set( $key, null );
 		}
 	}
+
+	/**
+	 * Doing a complete ACPu wipe with the red system cache.
+	 *
+	 * @hook clearCachePostProc
+	 *
+	 * @param array $cacheType The caching type.
+	 *
+	 * @return void
+	 */
+	public static function clearAPCuCache( array $cacheType )
+	{
+		if( class_exists( 'TYPO3\\CMS\\Core\\Configuration\\ExtensionConfiguration' ) ) {
+			$aimeosConfiguration = GeneralUtility::makeInstance( 'TYPO3\\CMS\\Core\\Configuration\\ExtensionConfiguration' )
+				->get( 'aimeos' );
+		} else {
+			$aimeosConfiguration = unserialize( $GLOBALS['TYPO3_CONF_VARS']['EXT']['extConf']['aimeos'] );
+		}
```

Comment:
The `getExtConfig()` method offers a simple method for that which works for all TYPO3 versions:
https://github.com/aimeos/aimeos-typo3/blob/master/Classes/Base.php#L99

Reply:
Done.